.github/workflows/cpp.yml

```
@@ -274,11 +274,11 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - windows-2019
+          - windows-2022
         include:
-          - os: windows-2019
+          - os: windows-2022
             simd-level: AVX2
-            title: AMD64 Windows 2019 C++17 AVX2
+            title: AMD64 Windows 2022 C++17 AVX2
```

Nit: C++17 is the default now, no need to mention it
```suggestion
            title: AMD64 Windows 2022 AVX2
```